entrypoint.sh

```diff
@@ -2,3 +2,21 @@
 
 sh -c "/usr/local/bin/php-cs-fixer fix $*"
```

Comment:
as of now, the cs-fixing run on every comment when using the newly-added config mentioned in the readme above.

we should somehow check whether we run a issue-comment action which does not contain the keyword and dont run the fixing process in this events

Reply:
maybe it would make sense to run the fixer in dry-run mode, when only checking syntax (and not fixing). This might be faster?

Reply:
per https://help.github.com/en/articles/events-that-trigger-workflows it seems that `GITHUB_EVENT_NAME` might be a good indicator whether the action was triggered by `on: issue_comment` or `on: [push, pull_request]`

Reply:
did another shot in the dark and implemented the above suggestion. still untested